docs: Use correct publish directory for netlify deploy (#2741)
medusajs/medusa: docs/content/deployments/admin/deploying-on-netlify.md

![Choose Repository](https://res.cloudinary.com/dza7lstvk/image/upload/v1668001871/Medusa%20Docs/Netlify/D0r6Q1e_th5uei.png)

> A section titled "Basic build settings" will appear. There should be three inputs that display, with some default settings set. Ensure the fields are set as follows (you may need to change the publish directory):
> 
> ```
> Base directory: (leave blank)
> Build command: yarn build
> Publish directory: public
> ```

Reviewer: ```suggestion
In the "Basic build settings" section, make sure the fields have the following values:

- Base directory: leave empty
- Build command: yarn build
- Publish directory: public

```

Author: Thank you for taking the time to respond. I feel that "make sure the fields have the following values," then, indicating "leave empty" with no text decoration (parentheses or otherwise), indicates to a reader that the base directory variable should be set to `leave empty`, though of course most users will probably read the meaning of words and leave the field empty. Yet, I still strive for annoying levels of clarity in documentation, especially considering how many non primary-english speakers probably use this excellent software. What do you think?

Reviewer: Makes sense! Then you can change it to `(leave empty)`

> When you've finished setting the "Basic build settings" fields, click the “Show advanced” button, which is above the “Deploy site” button.

Reviewer: ```suggestion
Next, click the “Show advanced” button, which is above the “Deploy site” button.
```

Author: I agree the brevity here makes sense.

##### Last Step: Steps with Default Values

> For the rest of the steps, you can keep most the default values provided by Netlify and press the "Enter" key on your keyboard for each. There are the following exceptions:

Reviewer: ```suggestion
For the rest of the steps, you can keep most of the default values provided by Netlify, except for the following options:
```
